framework/pom.xml

```diff
@@ -217,21 +217,21 @@
         <scope>test</scope>
     </dependency>
     <dependency>
-        <groupId>org.easymock</groupId>
-        <artifactId>easymock</artifactId>
-        <version>2.5.2</version>
-        <scope>test</scope>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>5.12.1</version>
+      <scope>test</scope>
     </dependency>
     <dependency>
         <groupId>org.mockito</groupId>
         <artifactId>mockito-core</artifactId>
-        <version>5.16.1</version>
+        <version>5.17.0</version>
         <scope>test</scope>
     </dependency>
     <dependency>
-        <groupId>junit</groupId>
-        <artifactId>junit</artifactId>
-        <version>4.13.1</version>
+        <groupId>org.assertj</groupId>
```

Comment:
Why not use the junit5 assertions? Like https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html

Or is assertj standerized in felix development?

Reply:
Because the osgi-test Setup uses also assertj and hase more powerfull assertions to do assertions on OSGi Framework.

Reply:
Clear 👍